src/main/java/il/co/orgo/orgo/dto/ShiftsOptionsDTO.java

```diff
@@ -0,0 +1,36 @@
+package il.co.orgo.orgo.dto;
+
+import il.co.orgo.orgo.model.ShiftsOptions;
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+
+import java.util.Date;
+
+@Data
+public class ShiftsOptionsDTO {
+
+    private Long userId;
+
+    private Long shiftId;
+
```

Comment:
This logic of creating an entity from DTO can be done as part of entity object creation. Just make a constructor inside the entity class. 
Then the logic of entity is in entity class and not in the DTO class. toShiftOptionsDTO is in correct place